package.json

```diff
@@ -48,7 +51,9 @@
     "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.3.0",
     "serve": "^11.1.0",
-    "webpack-manifest-plugin": "^2.0.4"
+    "webpack-manifest-plugin": "^2.0.4",
```

Comment:
are you using cookies?

Reply:
no, I installed by mistake I am removing it right now